packages/webapp/package.json

```diff
@@ -34,10 +34,10 @@
         "zustand": "^4.3.9"
     },
     "scripts": {
-        "start": "react-scripts start",
-        "build": "react-scripts build",
-        "test": "react-scripts test",
-        "eject": "react-scripts eject",
+        "start": "DISABLE_ESLINT_PLUGIN=true react-scripts start",
+        "build": "DISABLE_ESLINT_PLUGIN=true react-scripts build",
+        "test": "DISABLE_ESLINT_PLUGIN=true react-scripts test",
+        "eject": "DISABLE_ESLINT_PLUGIN=true react-scripts eject",
```

Comment:
upgrade of the packages created conflict with the CRA configuration, we are not using this feature anyway